test/unit/search-bar-test.spec.js

```diff
@@ -41,13 +41,13 @@ describe('Test Search Bar : ', () => {
       done();
     }).catch( e => { console.log(e.toString()) });
   });
-  it('Expect search button icon to be "search"', (done) => {
-    component.create(bootstrap).then( () => {
-      const element = document.querySelector('#search-bar_search-button-icon');
-      expect(element.innerHTML).toEqual('search');
-      done();
-    }).catch( e => { console.log(e.toString()) });
-  });
+  // it('Expect search button icon to be "search"', (done) => {
+  //   component.create(bootstrap).then( () => {
+  //     const element = document.querySelector('#search-bar_search-button-icon');
+  //     expect(element.innerHTML).toEqual('search');
+  //     done();
+  //   }).catch( e => { console.log(e.toString()) });
+  // });
```

Comment:
@Rodriguez-Carlos-BAH I'm not sure what to do with tests for elements we have drastically changed.